app/assets/javascripts/discourse/app/services/composer.js

```diff
@@ -1785,6 +1794,7 @@ export default class ComposerService extends Service {
 
   clearLastValidatedAt() {
     this.set("lastValidatedAt", null);
+    this.appEvents.trigger("composer-service:clear-last-validated-at");
```

Comment:
Should this be: 

```suggestion
    this.appEvents.trigger("composer-service:last-validated-at-cleared");
```

Since it's an event, it feels more accurate.